src/oca_pre_commit_hooks/checks_odoo_module_xml.py

```diff
@@ -378,5 +380,6 @@ def check_xml_oe_structure(self):
             for xpath_node in self.xpath_oe_structure_woid(manifest_data["node"]):
                 self.checks_errors["xml-oe-structure-missing-id"].append(
                     f'{manifest_data["filename_short"]}:{xpath_node.sourceline} '
-                    f"Consider removing the class 'oe_structure' or adding an id to the tag"
+                    "Consider removing the class 'oe_structure' or adding a proper id to the tag. The id must contain "
+                    "'oe_structure'"
```

Comment:
Even though current message is good, I think it could be improved, i.e.
- suggest including template ID in div ID. That way, we would reduce chances of collisions, people could be tempted to set `oe_structure_1`, which could be the same as other template
- The message explains there are two alternatives, it would be great to mention how to decide which one to take